context/Authentication.tsx

```diff
@@ -4,6 +4,7 @@ import { graphqlClient } from '@/api'
 import { signinWithEmailAndPasswordQuery } from '@/graphql/queries/user'
 import { useQueryClient } from '@tanstack/react-query'
 import { useCreateUser } from '@/hooks/mutation/user'
+import { useRouter } from 'next/router'
```

Comment:
could please separate these imports according to installed or build-in lib and custom or user-defined components/libs

like this: 
```
import { useQueryClient } from '@tanstack/react-query'
import { useRouter } from 'next/router'
import React, { createContext, useCallback, useContext, useMemo } from 'react'

import { graphqlClient } from '@/api'
import { CreateUserData } from '@/gql/graphql'
import { signinWithEmailAndPasswordQuery } from '@/graphql/queries/user'
import { useCreateUser } from '@/hooks/mutation/user'
```

Then it is ready to be merged!

Reply:
Okay👍